tests/sentry/relay/test_config.py

```diff
@@ -56,6 +56,18 @@ def test_get_project_config(default_project, insta_snapshot, full):
     insta_snapshot(cfg)
 
```

Comment:
I think it would be nice to have 1 negative test. 
E.g. when DynamicSampling raise an exception we catch it.